src/app/app.component.ts

```diff
@@ -442,6 +444,8 @@ export class CopayApp {
         }
       });
     }
+
+    await this.addressBookProvider.migrateOldContacts();
```

Comment:
We don't need to wait for this. It's safe to remove await in order to do the process asynchronous.